dotnet/src/webdriver/BiDi/Communication/Broker.cs

```diff
@@ -205,21 +212,14 @@ private async Task<EmptyResult> ExecuteCommandCoreAsync<TCommand>(TCommand comma
         where TCommand : Command
     {
         command.Id = Interlocked.Increment(ref _currentCommandId);
-
         var tcs = new TaskCompletionSource<EmptyResult>(TaskCreationOptions.RunContinuationsAsynchronously);
-
         var timeout = options?.Timeout ?? TimeSpan.FromSeconds(30);
-
         using var cts = new CancellationTokenSource(timeout);
-
         cts.Token.Register(() => tcs.TrySetCanceled(cts.Token));
-
         _pendingCommands[command.Id] = new(command.Id, command.ResultType, tcs);
 
-        var data = JsonSerializer.SerializeToUtf8Bytes(command, typeof(TCommand), _jsonSerializerContext);
-
+        var data = JsonSerializer.SerializeToUtf8Bytes(command, typeof(TCommand), _jsonSerializerOptions);
```

Comment:
Now that we are passing in options instead of context, we can make this strongly typed

```suggestion
        var data = JsonSerializer.SerializeToUtf8Bytes(command, _jsonSerializerOptions);
```